app/models/concerns/indexable.rb

```diff
@@ -104,8 +104,8 @@ def query(query, options={})
       # enable cursor-based pagination for DOIs
       if self.name == "Doi" && options.dig(:page, :cursor).present?
         from = 0
-        search_after = [options.dig(:page, :cursor)]
-        sort = [{ _id: { order: 'asc' }}]
+        search_after = Array.wrap(options.dig(:page, :cursor))
```

Comment:
Consider doing the base64decode here.

Reply:
I did originally put the decode here, but I felt that the query here doesn't really know it's DOI's and I feel like it makes it more in-line with the other cursors if it just takes what is being told is the cursor.
Additionally one of the reasons it doesn't work so well is that we have to support the cursor of page[cursor]=1 - With the logic handling in dois, it actually means that the page[cursor] gets set to an empty [] due to 1 not decoding properly, this then means it wont try and do this logic.

I could perhaps try and re-work it so we do an additional check here.

Reply:
Makes sense. 

We can change the logic of when cursor-based pagination is triggered, doesn't have to be that a page[cursor] parameter is present. And we might need some error handling for cursors that don't make sense, including `1`.